server/src/schema/event/eventType.ts

```diff
@@ -640,6 +646,72 @@ export const eventType = builder.prismaObject('TumiEvent', {
         return currentRegistrationNum < parent.organizerLimit;
       },
     }),
+    mailTemplate: t.string({
+      resolve: async (event, args, context) => {
+        const participatedText = marked(event.participantText);
+
+        const [icon, style] = (event.icon ?? '').split(':');
+        const iconUrl = `https://img.icons8.com/${style ?? 'fluency'}/64/${
+          icon ?? 'cancel-2'
+        }.png?token=9b757a847e9a44b7d84dc1c200a3b92ecf6274b2`;
+
+        const date = DateTime.fromJSDate(event.start).setLocale('en-US');
+        const intro = `Hi,<br/>thank you for signing up for the event on ${date.weekdayLong}!`;
+
+        let template = fs
+          .readFileSync(__dirname + '\\mailTemplate.mjml', 'utf8')
+          .replaceAll('%title%', event.title)
```

Comment:
We should maybe think about including handlebars, especially if more mails will follow